### PR TITLE
Add metrics collection and metrics-based dashboards info to Redis Otel

### PR DESCRIPTION
**Pull request by yasar-sumologic**
…l app

## Purpose of this pull request

Changes to redis-otel app doc after changes in it to add metric collection and metric-based dashboards

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/databases/opentelemetry/redis-opentelemetry.md [hunk 1]
```diff
@@ -30,10 +33,17 @@ The following are [Fields](/docs/manage/fields/) that will be created as part of
 * **`db.cluster.name`**. User configured. Enter a name to identify this Redis cluster. This cluster name will be shown in the Sumo Logic dashboards.
 * **`db.system`**. Has fixed value of redis.
 * **`deployment.environment`**. User configured. This is the deployment environment where the Redis cluster resides. For example: dev, prod or qa.
+* **`webengine.node.name`**. Has the value of host name of the machine which is being monitored
```
Comment (sumoanema) [inline docs/integrations/databases/opentelemetry/redis-opentelemetry.md:36]: this should be db.node.name ?

File: docs/integrations/databases/opentelemetry/redis-opentelemetry.md [hunk 2]
```diff
@@ -70,6 +80,11 @@ OpenTelemetry works with a [configuration](https://opentelemetry.io/docs/collect
 
 In this step, you will configure the yaml file required for Redis Collection.
 
+Below are the inputs required:
+
+- **`endpoint (no default)`**. The hostname and port of the Redis instance, separated by a colon..
```
Comment (sumoanema) [inline docs/integrations/databases/opentelemetry/redis-opentelemetry.md:85]: Can you give an example for clarity here

Comment (yasar-sumologic) [reply docs/integrations/databases/opentelemetry/redis-opentelemetry.md]: fixed it.

File: docs/integrations/databases/opentelemetry/redis-opentelemetry.md (hunk 1 above)
Comment (yasar-sumologic) [reply docs/integrations/databases/opentelemetry/redis-opentelemetry.md]: fixed it

File: docs/integrations/databases/opentelemetry/redis-opentelemetry.md [hunk 3]
```diff
@@ -171,3 +198,37 @@ Use this dashboard to:
...
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/Redis-Cluster-Overview.png' alt="Redis Cluster Overview dashboards" />
+
+### Command Stats Delta
+
+The **Redis - Command Stats Delta** dashboard provides an at-a-glance view of commands running on your Redis server.
+
+Use this dashboard to review String Commands, Key Commands, Hash Commands, Sets Commands, Sorted Sets Commands, List Commands, Transaction Commands, and Scripting Commands.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/Redis-Command-Stats-Delta.png' alt="Redis Command Stats Delta dashboards" />
```
Comment (jpipkin1) [inline docs/integrations/databases/opentelemetry/redis-opentelemetry.md:232]: This image is not rendering in my local build. Can you verify that the URL is correct?

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@yasar-sumologic, I made some minor changes (see commit "[Updates from review](https://github.com/SumoLogic/sumologic-documentation/pull/4286/commits/a35d4b8fade04be615c762031c7a6808eca66efa)").

**Review by jpipkin1 — APPROVED** [review]

**Comment by jpipkin1** [comment]
@yasar-sumologic, can we publish this PR now, or do we have to wait until the feature is released?

**Comment by yasar-sumologic** [comment]
> @yasar-sumologic, can we publish this PR now, or do we have to wait until the feature is released?

Yes we will have to wait until the feature is released

File: docs/integrations/databases/opentelemetry/redis-opentelemetry.md [hunk 4]
```diff
@@ -13,27 +13,36 @@ import TabItem from '@theme/TabItem';
...
 
-The following are [Fields](/docs/manage/fields/) that will be created as part of the Redis App install if not already present.
+## Creating fields in Sumo Logic for Redis
+
+The following are [fields](/docs/manage/fields/) that will be created as part of the Redis App install if not already present.
 
 * **`db.cluster.name`**. User configured. Enter a name to identify this Redis cluster. This cluster name will be shown in the Sumo Logic dashboards.
 * **`db.system`**. Has fixed value of redis.
 * **`deployment.environment`**. User configured. This is the deployment environment where the Redis cluster resides. For example: dev, prod or qa.
+* **`db.node.name`**. Has the value of host name of the machine which is being monitored
```
Comment (kimsauce) [inline docs/integrations/databases/opentelemetry/redis-opentelemetry.md:35]:
```suggestion
* **`db.node.name`**. Has the value of host name of the machine which is being monitored.
```

File: docs/integrations/databases/opentelemetry/redis-opentelemetry.md [hunk 5]
```diff
@@ -13,27 +13,36 @@ import TabItem from '@theme/TabItem';
...
 * **`sumo.datasource`**. Has fixed value of redis.
 
 ## Prerequisites
 
+### For metrics collection
+
+The Redis INFO command returns information and statistics about a Redis server (see Redis [INFO](https://redis.io/commands/info) for details). The Redis receiver extracts values from the result and converts them to open telemetry metrics.
```
Comment (kimsauce) [inline docs/integrations/databases/opentelemetry/redis-opentelemetry.md:42]:
```suggestion
The Redis INFO command returns information and statistics about a Redis server (see Redis [INFO](https://redis.io/commands/info) for details). The Redis receiver extracts values from the result and converts them to OpenTelemetry metrics.
```

File: docs/integrations/databases/opentelemetry/redis-opentelemetry.md [hunk 6]
```diff
@@ -70,12 +79,20 @@ OpenTelemetry works with a [configuration](https://opentelemetry.io/docs/collect
...
+- **Redis logs Path**. Enter the path to the log file for your Redis instance.
+
 The log file path configured to capture redis logs must be given here. The files are typically located in `/var/log/redis/redis-server.log`. If you are using a customized path, check the [`redis.conf`](https://download.redis.io/redis-stable/redis.conf) file for this information.
 
 You can add any custom fields which you want to tag along with the data ingested in Sumo. Click on the **Download YAML File** button to get the yaml file.
 
 <img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/Redis-YAML.png' style={{border:'1px solid gray'}} alt="Configuration" />
 
+:::note
+  Each Redis replica requires an independent sumo opentelemetry collector to be installed for effective log and metric monitoring.
```
Comment (kimsauce) [inline docs/integrations/databases/opentelemetry/redis-opentelemetry.md:94]:
```suggestion
  Each Redis replica requires an independent Sumo Logic OpenTelemetry collector to be installed for effective log and metric monitoring.
```

Comment (yasar-sumologic) [reply docs/integrations/databases/opentelemetry/redis-opentelemetry.md]: fixed it.

File: docs/integrations/databases/opentelemetry/redis-opentelemetry.md (hunk 5 above)
Comment (yasar-sumologic) [reply docs/integrations/databases/opentelemetry/redis-opentelemetry.md]: fixed it.

File: docs/integrations/databases/opentelemetry/redis-opentelemetry.md (hunk 4 above)
Comment (yasar-sumologic) [reply docs/integrations/databases/opentelemetry/redis-opentelemetry.md]: fixed it.

**Review by kimsauce — APPROVED** [review]

**Comment by jpipkin1** [comment]
@yasar-sumologic, can this PR be merged now, or does it have to wait for a feature release?

**Comment by yasar-sumologic** [comment]
> @yasar-sumologic, can this PR be merged now, or does it have to wait for a feature release?

Yes @jpipkin1  we can merge this PR

**Pull request merged**